tests/images/run.sh

```diff
@@ -41,3 +41,5 @@ done
 for png_filename in tests/images/imagenet_299/*.png; do
   ./bin/image-classifier "$png_filename" -image_mode=0to1 -m=googlenet_v4_slim/googlenet_v4_slim.onnx -model_input_name=input:0 -image_layout=NHWC -label_offset=1 "$@"
 done
+#Quantized Resnet50 Caffe2 model test
+./bin/image-classifier tests/images/imagenet/*.png -image_mode=0to1 -m=quant_resnet50 -model_input_name=gpu_0/data_0 -use-imagenet-normalization "$@"
```

Comment:
where do we get model from? Likely we need to update utils script for downloading the model

Reply:
Yes, the download_caffe2_models.sh is included in this PR.

Reply:
missed that :(